app/src/androidTest/java/com/example/crosswordToLearn/Utils.kt

```diff
@@ -148,7 +149,7 @@ fun waitForCondition(reason: String, condition: Callable<Boolean>, timeout: Long
 }
 
 fun waitForView(
-    viewMatcher: Matcher<View>, timeout: Long = 10000,
+    viewMatcher: Matcher<View>, timeout: Long = 30000,
```

Comment:
Do we need this change?

Reply:
Because I was advised to increase it and it helps to pass tests https://github.com/actions/virtual-environments/issues/2741#issuecomment-800805565

Reply:
Ok so it should be the same constant as for other timeouts.